### PR TITLE
Enable the rendering pipeline to be configured per-instance

### PR DESCRIPTION
**Pull request by jcoyne**

### CODE REVIEW

File: app/presenters/blacklight/rendering/pipeline.rb [hunk 1]
```diff
@@ -2,27 +2,42 @@
 
 module Blacklight
   module Rendering
-    # The field rendering pipeline
+    # The field rendering pipeline.
+    # This takes a field and it's values and transforms them through a list of
```
Comment (mjgiarlo) [inline app/presenters/blacklight/rendering/pipeline.rb]: `s/it's/its/`

File: app/presenters/blacklight/rendering/pipeline.rb [hunk 2]
```diff
@@ -2,27 +2,42 @@
...
+      class_attribute :operations, instance_accessor: false
 
       # The ordered list of pipeline operations
       self.operations = [HelperMethod, LinkToFacet, Microdata, Join]
 
-      def initialize(values, config, document, context, options)
+      # @param values [Array] the values for the field
+      # @param config [Blacklight::Configuration::Field] the field's configuration
+      # @param document [SolrDocument] the document
+      # @param context [Object] an execution context, used to execute the  helper method in.
```
Comment (mjgiarlo) [inline app/presenters/blacklight/rendering/pipeline.rb]: Trivial: two spaces between `the` and `helper`

**Comment by jcoyne** [comment]
Thanks for the keen eye @mjgiarlo!

**Comment by mjgiarlo** [comment]
@jcoyne Build went red. Restarted that build. Feel free to merge when this is green. :heavy_check_mark:

**Pull request merged**